src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/01-create-book-table.yml
+  - include:
+      file: db/changelog/changes/02-added-is-deleted-column-to-books.yml
+  - include:
+      file: db/changelog/changes/03-added-deleted-column-to-books.yml
+  - include:
+      file: db/changelog/changes/04-removed-is-deleted-column-from-books.yml
```

Comment:
always make sure each file ends with exactly one empty line (won't be displayed on GH), if GH shows red sign, it means there's none
![image](https://github.com/bohdan2704/bookstore/assets/49767049/5c5d2980-94d8-46a4-be8d-10b46627efe8)

```suggestion
      file: db/changelog/changes/04-removed-is-deleted-column-from-books.yml
      
```